Core/include/Acts/Propagator/NavigatorStatistics.hpp

```diff
@@ -0,0 +1,21 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <cstddef>
+
+namespace Acts {
+
+struct NavigatorStatistics {
+  std::size_t nRenavigations = 0;
```

Comment:
Should we explain what this counts in a comment? If yes, this applies also to the other statistics structs.